apps/meteor/tests/end-to-end/api/livechat/contacts.ts

```diff
@@ -632,4 +632,54 @@ describe('LIVECHAT - contacts', () => {
 			expect(res.body.errorType).to.be.equal('invalid-params');
 		});
 	});
+
+	describe('Contact Channels', () => {
+		let visitor: ILivechatVisitor;
+
+		beforeEach(async () => {
+			visitor = await createVisitor();
+		});
+
+		afterEach(async () => {
+			await deleteVisitor(visitor.token);
+		});
+
+		it('should add a channel to a contact when creating a new room', async () => {
```

Comment:
Any chance of having tests with other entry points from which a visitor can be created? (the SMS endpoint for example).